pagopa/io-app
# fix: [IOPID-498] Added function to set language on startup

## Short description
This pr fixes a regression that prevented the correct language setting found in the remote profile.

## List of changes proposed in this pull request
- ts/sagas/preferences.ts: added function `setLanguageFromProfileIfExists`
- ts/sagas/startup.ts: added call to the function

## How to test
1) Run dev server and put the simulator in english.
2) Install the app

The app should be in English until login.
After login, the app should be set in Italian language as the dev server (no first onboarding) replies "it" as preferred language
